x-pack/plugins/security_solution/common/detection_engine/diffable_rule_to_rule_response.ts

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type * as t from 'io-ts';
```

Comment:
This file in general can have some love unit tests, maybe it's oke to add after FF